service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -600,4 +604,16 @@ private ApiCloneWorkspaceResult fetchCloneWorkspaceResult(
     }
     return propertyMap;
   }
+
+  private AuthenticatedUserRequest getPetRequest(UUID workspaceId) {
```

Comment:
Same issue here, it's valid to clone a workspace which does not have a GCP cloud context.